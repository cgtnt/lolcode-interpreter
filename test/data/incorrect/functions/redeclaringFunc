HAI

HOW IZ I FOO
  FOUND YR 1
IF U SAY SO

HOW IZ I FOO
  FOUND YR 2
IF U SAY SO

KTHXBYE
