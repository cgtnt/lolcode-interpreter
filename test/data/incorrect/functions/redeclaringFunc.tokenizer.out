@1:T_BEGIN:HAI-@3:T_COMMAND_TERMINATOR:
-@3:T_FUNC_BEGIN:HOW IZ I-@3:T_T_IDENTIFIER:FOO-@4:T_COMMAND_TERMINATOR:
-@4:T_RETURN_VAL:FOUND YR-@4:T_T_INT:1-@5:T_COMMAND_TERMINATOR:
-@5:T_FUNC_END:IF U SAY SO-@7:T_COMMAND_TERMINATOR:
-@7:T_FUNC_BEGIN:HOW IZ I-@7:T_T_IDENTIFIER:FOO-@8:T_COMMAND_TERMINATOR:
-@8:T_RETURN_VAL:FOUND YR-@8:T_T_INT:2-@9:T_COMMAND_TERMINATOR:
-@9:T_FUNC_END:IF U SAY SO-@11:T_COMMAND_TERMINATOR:
-@11:T_END:KTHXBYE-@12:T_COMMAND_TERMINATOR:
-@12:T_EOF:
